docs/jwst/outlier_detection/arguments.rst

```diff
@@ -76,3 +76,21 @@ that control the behavior of the processing:
 
   For example, if set to ``0.5``, only resampled images that use less than half
   the available memory can be created.
+
+
+Step Arguments for IFU data
+===========================
+The `outlier_detection` step for IFU data has the following optional arguments
+that control the behavior of the processing:
+
+``--kernel_size`` (string, default='7 7')
+  The size of the kernel to use to normalize the pixel differences. The kernel size
+  must only contain odd values. 
+
+``--threshold_percent`` (float, default=99.8)
+The threshold (in percent) of the normalized minimum pixel difference used to identify bad pixels.
```

Comment:
Might need to indent these 2 lines of text (91+92) to match the indent of the start of text in line 90, in order to make docs build correctly.